text/0100-sensitive-data-handling.md

```diff
@@ -0,0 +1,103 @@
+#  Sensitive Data Handling
+
+By default, OpenTelemetry libraries never capture potentially-sensitive data, except for the full URL.
```

Comment:
What are "OpenTelemetry libraries"? Are these the API + SDK implementations or instrumentations/integrations that are provided by the OpenTelemetry project?

Reply:
This would include any current and future auto-instrumenter (like Java, .NET, node.js) as well as any manually wrapped libraries(e.g., in a contrib package).

Reply:
I see. It would make sense to explicitly mention the scope to which this OTEP and its design goals apply to in the document.